src/serialization/cnpy.cc

```diff
@@ -109,7 +109,7 @@ std::string dtype_descr(const TBlob& blob) {
     case mshadow::kUint64:
       return "'" MXNET_BYTEORDER "u8'";
     case mshadow::kBfloat16:
-      return "[('bfloat16', '" MXNET_BYTEORDER "u2')]";
+      return "'" MXNET_BYTEORDER "bfloat16'";
```

Comment:
other valuess here seems to safe the space. What about bf16 ? or rather according to convention bf2 or b2

Reply:
Agree, It shouldn't be dependant on order. So "b2" seems to me better candidate